pkgs/development/python-modules/lmfit/default.nix

```
@@ -50,21 +48,21 @@ buildPythonPackage rec {
   ];
 
   nativeCheckInputs = [
+    pytestCheckHook
+    pytest-cov-stub
     matplotlib
     pandas
-    pytest-cov-stub
-    pytestCheckHook
   ];
 
   pythonImportsCheck = [ "lmfit" ];
 
   disabledTests = [ "test_check_ast_errors" ];
 
-  meta = with lib; {
+  meta = {
     description = "Least-Squares Minimization with Bounds and Constraints";
     homepage = "https://lmfit.github.io/lmfit-py/";
     changelog = "https://github.com/lmfit/lmfit-py/releases/tag/${version}";
-    license = licenses.bsd3;
-    maintainers = with maintainers; [ nomeata ];
+    license = lib.licenses.bsd3;
+    maintainers = with lib.maintainers; [ nomeata ];
```

Maybe it would be more honest to
```suggestion
    maintainers = with lib.maintainers; [ ];
```
since I am not longer using this library. Do you want to add yourself?

---

OK sounds fair to me.